modules/spring/src/main/java/org/apache/ignite/IgniteSpringBean.java

```diff
@@ -93,8 +95,7 @@
  * </pre>
  * <p>
  */
-public class IgniteSpringBean implements Ignite, DisposableBean, SmartInitializingSingleton,
-    ApplicationContextAware, Externalizable {
+public class IgniteSpringBean extends AbstractLifecycleBean implements Ignite, ApplicationContextAware, Externalizable {
```

Comment:
Sorry, I wanted to make everything a bit consistent but I didn't take in mind backward compatibility. We need SmartLifeCycle to start bean automatically at the time that the containing ApplicationContext gets refreshed.Otherwise, we need to call the start method ourselves.

Reply:
I have reverted my IgniteSpringBean changes.